plugins/modules/cloud/univention/udm_dns_record.py

```diff
@@ -124,14 +148,30 @@ def main():
     changed = False
     diff = None
 
+    workname = name
+    if type == 'ptr_record':
+        if not HAVE_IPADDRESS:
+            module.fail_json(msg="This module requires the 'ipaddress' python module to manage PTR records.")
```

Comment:
```suggestion
            module.fail_json(msg=missing_required_lib('ipaddress'))
```
(needs `from ansible.module_utils.basic import missing_required_lib`).